src/assessments/custom-widgets/custom-widgets-column-renderer.tsx

```diff
@@ -55,94 +55,90 @@ function makeFlatDesignPatternString(patterns: DesignPattern[]): string {
 }
 
 const roleToDesignPatternsMapping: DictionaryStringTo<DesignPattern[]> = {
-    alert: [{ designPattern: 'Alert', URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#alert' }],
+    alert: [{ designPattern: 'Alert', URL: 'https://www.w3.org/WAI/ARIA/apg/patterns/alert' }],
     alertdialog: [
         {
-            designPattern: 'Alert or Message Dialog',
-            URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog',
+            designPattern: 'Alert and Message Dialogs',
```

Comment:
This should be Alert or Message (in this case the design pattern can be one or the other but not both

Reply:
Actually, copied the title from w3.
Updating..